test-prow-e2e.sh

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env bash
+
+set -eExuo pipefail
+
+NAMESPACE=""            # TODO
+MCG_MS_CONSOLE_IMAGE="" # TODO
```

Comment:
@v-harihar Do we have a final name for the image being used? IIRC for ODF it's just `odf-console`, so should it be `mcg-ms-console` in our case?